hash/megadriv.xml

```diff
@@ -25099,6 +25099,38 @@ Notice that these are not working on real hardware due to bugged code with VDP i
 		</part>
 	</software>
 
+	<software name="sonic2p9" cloneof="sonic2">
+		<description>Sonic the Hedgehog 2 (World, prototype 19920821)</description>
+		<year>1992</year>
+		<publisher>Sega</publisher>
+		<part name="cart" interface="megadriv_cart">
+			<dataarea name="rom" width="16" endianness="big" size="1048576">
+				<rom name="md sonic 2 0 21.8 alpha.bin" size="131072" crc="53c60603" sha1="70e3c828a8553d44d9d7713a61a667d492cd059f"/>
```

Comment:
Please don’t omit the explicit offset if there are multiple ROM load directives.